src/main/scala/com/itechart/ny_accidents/Application.scala

```diff
@@ -53,9 +53,9 @@ object Application extends App {
   logger.info("Accidents count per hour for each phenomenon metric calculated")
 
```

Comment:
Extra blank lines. If you want to make your code more readable and highlight some code in this way it's more desired to extract this piece of code into a separate method as it's self described/explanatory.

Reply:
Yes, Application will be simplified in next sprint in a separate task